modules/charon-core/src/main/java/org/wso2/charon3/core/encoder/JSONDecoder.java

```diff
@@ -753,6 +753,18 @@ public ArrayList<PatchOperation> decodeRequest(String scimResourceString) throws
             JSONObject decodedJsonObj = new JSONObject(new JSONTokener(scimResourceString));
             //obtain the Operations values
             JSONArray operationJsonList = (JSONArray) decodedJsonObj.opt(SCIMConstants.OperationalConstants.OPERATIONS);
+
+            //check if operationJsonList is null
+            if (operationJsonList == null) {
+
```

Comment:
We can remove new lines Line 759, 764

Reply:
Fixed by https://github.com/wso2/charon/pull/406/commits/14c2ca2e08db26274028bd2d44ce71ae52aca695